### PR TITLE
Add Disclaimer Statement

### PR DESCRIPTION
**Pull request by inceptionev**
added liability statement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,458 +1,478 @@
...
-| mask4   | 1 | Allegro | Tyvek Full-Face Hood | $23 | [Z][9amzn] | example full-face hood* |
-| mask5   | 1 | Custom 3D print | Bayonet Inlet Adapter | $2 (material) | [3D][3d7] | Inlet Adapter for Bayonet-type masks |
-| mask6   | 1 | Custom 3D print | 6100/6200/6300 Outlet Plug | $1 (material) | [3D][3d8] | Plug for sealing the outlet valve of 6100/6200/6300 masks |
-| mask7   | 1 | 3M | Inlet Filter Gasket  | $2.01 | [G][1grgr] | Inlet Gasket used as part of above outlet sealing |
+# RespiraWorks PAPR Project
+
+![Overview photo](/assets/Overview-PAPR-Collage.JPG)
+
+***Important Disclaimer: This is a prototype device, offered free-of-charge as a public service during the COVID emergency, with no claim, guarantee, or warranty offered or implied of performance or suitablity for any purpose.  This device has not be independently certified or approved.  Use of this design or any derivatives thereof is entirely at your own risk.  By using any of these materials, you acknowledge that you have read and understood the [full disclaimer](#full-disclaimer) at the end of this document.  Please do not use these materials if you do not understand and accept these risks.***
```
Comment (curtiskline) [inline README.md]: "has not be independently" should be "has not been independently"

**Review by curtiskline — CHANGES_REQUESTED** [review]
Other than my one small comment, this looks good to me.

File: README.md (hunk 1 above)
Comment (inceptionev) [reply README.md]: Thanks for catching that, fixed now.

**Review by curtiskline — APPROVED** [review]
lgtm

**Pull request merged**